app/services/workflow_executions/status_service.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module WorkflowExecutions
+  # Service used to update the status of a WorkflowExecution
+  class StatusService < BaseService
+    CANCELATION_STATES = %w[
```

Comment:
Question: Should these state arrays belong to the API client?

Which strings mean which states is something that could be changed in a version update to the API. Added to a file like `lib/integrations/ga4gh_wes_api/v1/states.rb` might make sense.

Reply:
Yeah I'm thinking we will move them somewhere else. Just put them inside the service for now and can be updated later once we figure out where we want them. @ericenns thoughts?

Reply:
I ended up adding the states to the above module in adfc38e07a90093c73bc75f5a0f3707792a7900e